src/lib.rs

```diff
@@ -121,7 +121,8 @@ pub trait CurveAffine:
 
 /// An encoded elliptic curve point, which should essentially wrap a `[u8; N]`.
 pub trait EncodedPoint:
-    Sized + Send + Sync + AsRef<[u8]> + AsMut<[u8]> + Clone + Copy + 'static
+    Sized + Send + Sync + AsRef<[u8]> + AsMut<[u8]> + Clone + Copy
+        + PartialOrd + Ord + PartialEq + Eq + ::std::hash::Hash + 'static
```

Comment:
These bounds should now be applied to `GroupEncoding::Repr` and `UncompressedEncoding::Repr`. Also add `use core::hash::Hash` at the top of the file instead of the explicit form here.